presto-product-tests/src/main/java/io/prestosql/tests/iceberg/TestCreateTable.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.iceberg;
+
+import io.airlift.log.Logger;
+import io.prestosql.tempto.AfterTestWithContext;
+import io.prestosql.tempto.BeforeTestWithContext;
+import io.prestosql.tempto.ProductTest;
+import org.testng.annotations.Test;
+
+import static io.prestosql.tempto.assertions.QueryAssert.assertThat;
+import static io.prestosql.tempto.query.QueryExecutor.query;
+import static io.prestosql.tests.TestGroups.CREATE_TABLE;
+
+public class TestCreateTable
+        extends ProductTest
+{
+    private static final String ICEBERG_TABLE_NAME1 = "iceberg.ice_tables.nested_types";
+    private static final String ICEBERG_TABLE_NAME2 = "iceberg.ice_tables.nested_types2";
+    private static final String ICEBERG_TABLE_SCHEMA = "iceberg.ice_tables";
+
+    @BeforeTestWithContext
+    public void createObjects()
+    {
+        // Drop Iceberg tables and schema
+        try {
+            query("DROP TABLE IF EXISTS " + ICEBERG_TABLE_NAME1);
+            query("DROP TABLE IF EXISTS " + ICEBERG_TABLE_NAME2);
+        }
+        catch (Exception e) {
```

Comment:
We shouldn't catch exceptions like this in tests, as we want any such exception to fail the test.

Reply:
Got it.